wgpu-hal/src/vulkan/instance.rs

```diff
@@ -557,6 +559,18 @@ impl crate::Instance<super::Api> for super::Instance {
             layers
         };
 
+        #[cfg(target_os = "android")]
+        let platform_version = {
```

Comment:
How about android_platform_version just to make it clear that this value is meaningless on other platforms.

Reply:
yeah, sounds good to me; maybe `android_sdk_version` to help clarify that it won't have a value like 10 for Android 10 but rather will be based on these SDK version codes: https://developer.android.com/reference/android/os/Build.VERSION_CODES